solutions/standard/variables.tf

```diff
@@ -115,8 +122,212 @@ variable "en_key_name" {
+  nullable    = true
+  default     = null
+  description = "The CRN of an existing Cloud Object Storage instance. If not supplied, a new instance will be created."
+}
+
+variable "create_cos_bucket" {
+  description = "Set as true to create a new Cloud Object Storage bucket"
+  type        = bool
+  default     = true
+}
+
+variable "existing_cos_bucket_name" {
+  type        = string
+  nullable    = true
+  default     = null
+  description = "The name of an existing bucket inside the existing Cloud Object Storage instance. If not supplied, a new bucket will be created."
+}
+
+variable "cos_destination_name" {
+  type        = string
+  description = "The name to give the IBM Cloud Object Storage destination which will be created for storage of failed delivery events."
+  default     = "COS Destination"
+}
+
+variable "cos_bucket_name" {
+  type        = string
+  description = "The bucket name in IBM cloud object storage instance. A bucket will not be created if a value is passed for `var.existing_cos_bucket_name`."
+  default     = "base-event-notifications-bucket"
+}
+
+variable "skip_en_cos_auth_policy" {
+  type        = bool
+  description = "Set to true to skip the creation of an IAM authorization policy that permits all Event Notification instances in the resource group to interact with your Cloud Object Storage instance."
+  default     = false
+}
+
+variable "cos_instance_name" {
+  type        = string
+  default     = "base-security-services-cos"
+  description = "The name to use when creating the Cloud Object Storage instance."
+}
+
+variable "cos_instance_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to Cloud Object Storage instance. Only used if not supplying an existing instance."
+  default     = []
+}
+
+variable "cos_instance_access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the Cloud Object Storage instance. Only used if not supplying an existing instance."
+  default     = []
+}
+
+variable "add_bucket_name_suffix" {
+  type        = bool
+  description = "Add random generated suffix (4 characters long) to the newly provisioned COS bucket name. Only used if not passing existing bucket. set to false if you want full control over bucket naming using the 'cos_bucket_name' variable."
+  default     = true
+}
+
+variable "cos_plan" {
+  description = "Plan to be used for creating cloud object storage instance. Only used if 'create_cos_instance' it true."
+  type        = string
+  default     = "standard"
+  validation {
+    condition     = contains(["standard", "lite", "cos-one-rate-plan"], var.cos_plan)
+    error_message = "The specified cos_plan is not a valid selection!"
+  }
+}
+
+variable "cross_region_location" {
+  description = "Specify the cross-regional bucket location. Supported values are 'us', 'eu', and 'ap'. If you pass a value for this, ensure to set the value of var.region and var.single_site_location to null."
```

Comment:
don't use `var.` in descriptions for DAs - it doesn't make sense in Projects land where the DA will be getting consumed :)